src/createServer.js

```diff
@@ -1,13 +1,67 @@
 'use strict';
 
-// const express = require('express');
-
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const express = require('express');
+  const fs = require('fs');
+  const path = require('path');
```

Comment:
you should add it only once, not for each function call, let's move it outside the function